msgpack/src/main/scala/fs2/data/msgpack/low/package.scala

```diff
@@ -18,11 +18,22 @@ package fs2
 package data
 package msgpack
 
-import low.internal.ItemParser
+import low.internal.{ItemParser, ItemSerializer, ItemValidator}
 
 /** A low-level representation of the MessagePack format.
   */
 package object low {
   def items[F[_]](implicit F: RaiseThrowable[F]): Pipe[F, Byte, MsgpackItem] =
     ItemParser.pipe[F]
+
+  /** Alias for `bytes(validated = true)`
```

Comment:
Where is `bytes` defined?

Reply:
Forgot to change this in [309569e](https://github.com/gnieh/fs2-data/pull/628/commits/309569e48961642af378989709a29637f377759f) :smile: